cli/wallet/multisig.go

```diff
@@ -0,0 +1,13 @@
+package wallet
+
+import "github.com/CityOfZion/neo-go/pkg/core/transaction"
+
+// TxWithSignatures contains partially signed transaction.
+type TxWithSignatures struct {
```

Comment:
Is it the same format neo-cli/neo-gui uses?

Reply:
No, I will change this to their format.

Reply:
Fixed.